cogs/birthday/birthday.py

```diff
@@ -433,12 +438,38 @@ async def getSelfBirthday(self, ctx: Context):
                 if month and day:
                     birthday = date(2020, month, day)
                     birthdayStr = "{0:%B} {0:%d}".format(birthday)
-                    await ctx.author.send(
+                    birthdayInfoMsg = (
                         f"{headerGood}: Your birthday is "
                         f"{spoiler(bold(birthdayStr, escape_formatting=False), escape_formatting=False)}."
                     )
-                    return
-
+                    try:
+                        await ctx.author.send(birthdayInfoMsg)
+                        return
+                    except discord.Forbidden:
+                        await ctx.send(
+                            f"{headerWarn}: I would like to DM you your birthday but it seeems that"
```

Comment:
Leave a trailing space here.
```suggestion
                            f"{headerWarn}: I would like to DM you your birthday but it seeems that "
```

Reply:
Fixed